src/events/eventActions.js

```diff
@@ -1,97 +1,94 @@
 /* @flow */
-import type { Auth, Dispatch, GetState, Action } from '../types';
+import type { Dispatch, GetState, Action } from '../types';
 import { pollForEvents, registerForEvents } from '../api';
 import { switchAccount } from '../account/accountActions';
 import { timeout } from '../utils/async';
 import eventToAction from './eventToAction';
 import eventMiddleware from './eventMiddleware';
+import { getAuth } from '../account/accountSelectors';
 
-import {
-  BATCH_ACTIONS,
-  REALM_INIT,
-  EVENT_REGISTERED,
-} from '../actionConstants';
+import { BATCH_ACTIONS, REALM_INIT, EVENT_REGISTERED } from '../actionConstants';
 
-const startEventPolling = (auth: Auth, queueId: number, eventId: number) =>
-  async (dispatch: Dispatch, getState: GetState) => {
-    let lastEventId = eventId;
-    let numFailures = 0;
+const startEventPolling = (queueId: number, eventId: number) => async (
```

Comment:
I don't think this formatting is an improvement. It is inconsistent.
A function definition on one line, then another one on another line seems quite logical (as it was originally).